README.md

```diff
@@ -21,12 +21,20 @@ Plugins listed here should be _stable_ and actively _maintained_. If you have is
 - Read the [creating plugins guide](https://github.com/asdf-vm/asdf/blob/master/docs/plugins-create.md)
 - Consider using our [Template](https://github.com/asdf-vm/asdf-plugin-template) which has the core functionality to tools published to GitHub releases and CI for GitHub/GitLab/CircleCI out of the box.
 
+### `asdf-community`
+
+If you're creating a new plugin consider creating it as part of the [`asdf-community`](https://github.com/asdf-community/.github) project. This is a separate community project with consolidated maintenance.
```

Comment:
Why? What's the benefit here for us?

Reply:
It was raised in #180 without objection and to also avoid year long conversations like #274

We can remove this section. Was just trying to close out older Issues.